### DISCUSSION TITLE
[REQ]

### DISCUSSION THREAD

**Discussion by ghost** [Chia-Network/chia-blockchain#3191] [General]
Hello, I will have a special and short question. 
I have 1 plot now and I have it drawn with k-32.
Chia tells me, that I will win in 6 years, so how do I know if I won with farming now?
Thank you, we are waiting for your help.

**Comment by maomingyang1314** [comment]
If you win, you will get HCX as a reward

**Reply by ghost** [reply]
But, I dont see reward is my wallet, whats min reward number ?
